### ISSUE TITLE
Faces of sampled point cloud from meshes

### ISSUE THREAD

**Issue by Hippogriff** [daavoo/pyntcloud#202]
How to get face id of the points sampled from meshes?

**Comment by Hippogriff** [comment]
Yeah. This is what I finally ended up doing. Poisson sampling should also be included. I will try implementing and will send PR.

**Comment by movvam** [comment]
> Yeah. This is what I finally ended up doing. Poisson sampling should also be included. I will try implementing and will send PR.

Making this its own feature request #239
